crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -626,6 +631,7 @@ impl Display for JsValue {
             JsValue::Argument(func_ident, index) => {
                 write!(f, "arguments[{}#{}]", index, func_ident)
             }
+            JsValue::Iterated(_, iterable) => write!(f, "for (let _ of {})", iterable),
```

Comment:
```suggestion
            JsValue::Iterated(_, iterable) => write!(f, "Iterated({})", iterable),
```


It need to look like an expression for formatting